doc/getting_started.md

```diff
@@ -0,0 +1,66 @@
+# Getting started

```

Comment:
I have followed the structure recommended. However, I have instead of "For more information...", I have included the line "Please refer the Get started guide for detailed setup instructions."